fhir-persistence/src/test/java/com/ibm/fhir/persistence/util/PartitionedSequentialKeyTest.java

```diff
@@ -0,0 +1,64 @@
+
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+package com.ibm.fhir.persistence.util;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotEquals;
+import static org.testng.Assert.assertTrue;
+
+import org.testng.annotations.Test;
+
+/**
+ * Unit test for PartitionedSequentialKey
+ */
+public class PartitionedSequentialKeyTest {
+    
+    private void safeSleep(long millis) {
+        try {
+            Thread.sleep(millis);
+        } catch (InterruptedException x) {
+            // NOP
+        }
+    }
```

Comment:
I see this used in so many places, can we have a single SleepUtil that does this code.